osu.Game/Localisation/FirstRunSetupOverlayStrings.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class FirstRunSetupOverlayStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.FirstRunSetupOverlay";
+
+        /// <summary>
+        /// "Get started"
+        /// </summary>
+        public static LocalisableString GetStarted => new TranslatableString(getKey(@"get_started"), @"Get started");
+
+        /// <summary>
+        /// "Click to resume first-run setup at any point"
+        /// </summary>
+        public static LocalisableString ClickToResumeFirstRunSetupAtAnyPoint => new TranslatableString(getKey(@"click_to_resume_first_run_setup_at_any_point"), @"Click to resume first-run setup at any point");
+
+        /// <summary>
+        /// "First-run setup"
+        /// </summary>
+        public static LocalisableString FirstRunSetup => new TranslatableString(getKey(@"first_run_setup"), @"First-run setup");
+
+        /// <summary>
+        /// "Setup osu! to suit you"
+        /// </summary>
+        public static LocalisableString SetupOsuToSuitYou => new TranslatableString(getKey(@"setup_osu_to_suit_you"), @"Setup osu! to suit you");
```

Comment:
was bugging me slightly

"setup" is a noun and "set up" is a verb

```suggestion
        /// "Set up osu! to suit you"
        /// </summary>
        public static LocalisableString SetupOsuToSuitYou => new TranslatableString(getKey(@"setup_osu_to_suit_you"), @"Set up osu! to suit you");
```

Reply:
I knew something felt off about this line.